miner/miner.go

```diff
@@ -42,13 +41,6 @@ func RestartMiner(bc *blockchain.BlockChain, b *blockchain.Block) {
 // TODO: Make Mine take an interface with a callback as an arguement.
 func Mine(bc *blockchain.BlockChain, b *blockchain.Block) *MiningResult {
 	setStart()
-	if valid, _ := bc.ValidBlock(b); !valid {
```

Comment:
The miner receives blocks from the `pool`, and the `app` orchestrates. No need to re-validate 👍